### :x: 1 Tests Failed:
| Tests completed | Failed | Passed | Skipped |
|---|---|---|---|
| 4231 | 1 | 4230 | 1 |
<details><summary>View the full list of 1 :snowflake: flaky test(s)</summary>

> <pre><code class="language-python">tests/entrypoints/options/pages/ManagedSiteChannels/ManagedSiteChannels.test.tsx &gt; ManagedSiteChannels &gt; sends a targeted sync request from row actions and reports backend failures</code></pre>
> **Flake rate in main:** 14.29% (Passed 12 times, Failed 2 times)
> <details><summary>Stack Traces | 9.44s run time</summary>
> 
> > <pre><code class="language-python">AssertionError: expected false to be true // Object.is equality
> > 
> > Ignored nodes: comments, script, style
> > &lt;html&gt;
> >   &lt;head /&gt;
> >   &lt;body
> >     style=&quot;&quot;
> >   &gt;
> >     &lt;div&gt;
> >       &lt;div
> >         class=&quot;space-y-6 p-6&quot;
> >       &gt;
> >         &lt;div
> >           class=&quot;mb-8&quot;
> >         &gt;
> >           &lt;div
> >             class=&quot;flex flex-col gap-2 md:flex-row md:items-start md:justify-between md:gap-4&quot;
> >           &gt;
> >             &lt;div
> >               class=&quot;flex items-center gap-3&quot;
> >             &gt;
> >               &lt;svg
> >                 aria-hidden=&quot;true&quot;
> >                 class=&quot;lucide lucide-layers h-6 w-6 text-blue-600 dark:text-blue-400&quot;
> >                 fill=&quot;none&quot;
> >                 height=&quot;24&quot;
> >                 stroke=&quot;currentColor&quot;
> >                 stroke-linecap=&quot;round&quot;
> >                 stroke-linejoin=&quot;round&quot;
> >                 stroke-width=&quot;2&quot;
> >                 viewBox=&quot;0 0 24 24&quot;
> >                 width=&quot;24&quot;
> >                 xmlns=&quot;http://www.w3.org/2000/svg&quot;
> >               &gt;
> >                 &lt;path
> >                   d=&quot;M12.83 2.18a2 2 0 0 0-1.66 0L2.6 6.08a1 1 0 0 0 0 1.83l8.58 3.91a2 2 0 0 0 1.66 0l8.58-3.9a1 1 0 0 0 0-1.83z&quot;
> >                 /&gt;
> >                 &lt;path
> >                   d=&quot;M2 12a1 1 0 0 0 .58.91l8.6 3.91a2 2 0 0 0 1.65 0l8.58-3.9A1 1 0 0 0 22 12&quot;
> >                 /&gt;
> >                 &lt;path
> >                   d=&quot;M2 17a1 1 0 0 0 .58.91l8.6 3.91a2 2 0 0 0 1.65 0l8.58-3.9A1 1 0 0 0 22 17&quot;
> >                 /&gt;
> >               &lt;/svg&gt;
> >               &lt;h2
> >                 class=&quot;text-2xl font-semibold text-left dark:text-dark-text-primary text-gray-900&quot;
> >               &gt;
> >                 managedSiteChannels:title
> >               &lt;/h2&gt;
> >             &lt;/div&gt;
> >             &lt;div
> >               class=&quot;flex shrink-0 flex-wrap items-center gap-3&quot;
> >             &gt;
> >               &lt;div
> >                 class=&quot;flex flex-wrap items-center gap-2&quot;
> >               &gt;
> >                 &lt;button
> >                   class=&quot;inline-flex items-center justify-center gap-2 whitespace-nowrap rounded-md text-sm font-medium transition-all disabled:pointer-events-none disabled:opacity-50 [&amp;_svg]:pointer-events-none [&amp;_svg:not([class*=&#x27;size-&#x27;])]:size-4 shrink-0 [&amp;_svg]:shrink-0 outline-none focus-visible:border-ring focus-visible:ring-ring/50 focus-visible:ring-[3px] aria-invalid:ring-destructive/20 dark:aria-invalid:ring-destructive/40 aria-invalid:border-destructive border bg-background shadow-xs hover:bg-accent hover:text-accent-foreground dark:bg-input/30 dark:border-input dark:hover:bg-input/50 h-9 px-4 py-2 has-[&gt;svg]:px-3&quot;
> >                   data-slot=&quot;button&quot;
> >                 &gt;
> >                   &lt;span&gt;
> >                     &lt;svg
> >                       aria-hidden=&quot;true&quot;
> >                       class=&quot;lucide lucide-refresh-ccw h-4 w-4&quot;
> >                       fill=&quot;none&quot;
> >                       height=&quot;24&quot;
> >                       stroke=&quot;currentColor&quot;
> >                       stroke-linecap=&quot;round&quot;
> >                       stroke-linejoin=&quot;round&quot;
> >                       stroke-width=&quot;2&quot;
> >                       viewBox=&quot;0 0 24 24&quot;
> >                       width=&quot;24&quot;
> >                       xmlns=&quot;http://www.w3.org/2000/svg&quot;
> >                     &gt;
> >                       &lt;path
> >                         d=&quot;M21 12a9 9 0 0 0-9-9 9.75 9.75 0 0 0-6.74 2.74L3 8&quot;
> >                       /&gt;
> >                       &lt;path
> >                         d=&quot;M3 3v5h5&quot;
> >                       /&gt;
> >                       &lt;path
> >                         d=&quot;M3 12a9 9 0 0 0 9 9 9.75 9.75 0 0 0 6.74-2.74L21 16&quot;
> >                       /&gt;
> >                       &lt;path
> >                         d=&quot;M16 16h5v5&quot;
> >                       /&gt;
> >                     &lt;/svg&gt;
> >                   &lt;/span&gt;
> >                   managedSiteChannels:toolbar.refresh
> >                 &lt;/button&gt;
> >                 &lt;button
> >                   class=&quot;inline-flex items-center justify-center gap-2 whitespace-nowrap rounded-md text-sm font-medium transition-all disabled:pointer-events-none disabled:opacity-50 [&amp;_svg]:pointer-events-none [&amp;_svg:not([class*=&#x27;size-&#x27;])]:size-4 shrink-0 [&amp;_svg]:shrink-0 outline-none focus-visible:border-ring focus-visible:ring-ring/50 focus-visible:ring-[3px] aria-invalid:ring-destructive/20 dark:aria-invalid:ring-destructive/40 aria-invalid:border-destructive border bg-background shadow-xs hover:bg-accent hover:text-accent-foreground dark:bg-input/30 dark:border-input dark:hover:bg-input/50 h-9 px-4 py-2 has-[&gt;svg]:px-3&quot;
> >                   data-slot=&quot;button&quot;
> >                 &gt;
> >                   &lt;span&gt;
> >                     &lt;svg
> >                       aria-hidden=&quot;true&quot;
> >                       class=&quot;lucide lucide-arrow-right-left h-4 w-4&quot;
> >                       fill=&quot;none&quot;
> >                       height=&quot;24&quot;
> >                       stroke=&quot;currentColor&quot;
> >                       stroke-linecap=&quot;round&quot;
> >                       stroke-linejoin=&quot;round&quot;
> >                       stroke-width=&quot;2&quot;
> >                       viewBox=&quot;0 0 24 24&quot;
> >                       width=&quot;24&quot;
> >                       xmlns=&quot;http://www.w3.org/2000/svg&quot;
> >                     &gt;
> >                       &lt;path
> >                         d=&quot;m16 3 4 4-4 4&quot;
> >                       /&gt;
> >                       &lt;path
> >                         d=&quot;M20 7H4&quot;
> >                       /&gt;
> >                       &lt;path
> >                         d=&quot;m8 21-4-4 4-4&quot;
> >                       /&gt;
> >                       &lt;path
> >                         d=&quot;M4 17h16&quot;
> >                       /&gt;
> >                     &lt;/svg&gt;
> >                   &lt;/span&gt;
> >                   &lt;sp...
> > 
> > - Expected
> > + Received
> > 
> > - true
> > + false
> > 
> >  ❯ .../pages/ManagedSiteChannels/ManagedSiteChannels.test.tsx:200:35
> >  ❯ runWithExpensiveErrorDiagnosticsDisabled node_modules/@.../dom/dist/config.js:47:12
> >  ❯ checkCallback node_modules/@.../dom/dist/wait-for.js:124:77
> >  ❯ Timeout.checkRealTimersCallback node_modules/@.../dom/dist/wait-for.js:118:16</code></pre>
> 
> </details>

</details>

To view more test analytics, go to the [Test Analytics Dashboard](https://app.codecov.io/gh/qixing-jk/all-api-hub/tests/fix%2Ftemp-window-fallback-reminder-gate-rendering)
<sub>📋 Got 3 mins? [Take this short survey](https://forms.gle/22i53Qa1CySZjA6c7) to help us improve Test Analytics.</sub>

--- a/src/features/AccountManagement/components/TempWindowFallbackReminderGate.tsx
+++ b/src/features/AccountManagement/components/TempWindowFallbackReminderGate.tsx
@@ -126,9 +126,7 @@ export function TempWindowFallbackReminderGate() {
   ])
 
   const isReminderDialogOpen =
-    hasConfirmedReminderIssue &&
-    !tempWindowFallbackReminder.dismissed &&
-    (isOpen || !hasShownInThisSession)
+    hasConfirmedReminderIssue && !tempWindowFallbackReminder.dismissed && isOpen
 
   const handleNeverRemind = async () => {
     await updateTempWindowFallbackReminder({ dismissed: true })

